atsign-foundation/atsign.dev-3.0
# fix: Added mobile support for panel shortcode

**- What I did**
- Changed how the panel shortcode layout looks for mobile.
**- How I did it**
- Added media query in CSS in _general.scss and changed certain properties to make it look better.
**- How to verify it**
- See changes in most recent commit.
- See the following screenshots: 
![image](https://user-images.githubusercontent.com/34658202/178334042-c683d2ed-9eda-4c00-8a69-78c5aaa9eb61.png)

**- Description for the changelog**
Added mobile support for panel shortcode
